### DISCUSSION TITLE
Non NVIDIA GPU

### DISCUSSION THREAD

**Discussion by antoninrat** [mitsuba-renderer/mitsuba3#749] [Q&A] [answered]
Hi, 

Is there any way to use Mitsuba3 on non-NVIDIA GPUs?
If not, is this on your roadmap for future developments?

Thanks,

**Comment by njroussel** [comment, marked as answer]
Hi @antoninrat 

Currently, Mitsuba heavily relies on OptiX which is locked to Nvidia drivers only.

We do have _some_ plans to _potentially_ support Metal (see [here](https://github.com/mitsuba-renderer/mitsuba3/discussions/180) for more details. However, HIP/Vulkan/etc. are not on our own roadmap.
